src/sbt-test/sbt-pack/exclude-test-config/build.sbt

```diff
@@ -0,0 +1,43 @@
+scalaVersion in ThisBuild := "2.12.6"
+
+val commonSettings = Seq(
+  scalaVersion := "2.12.6",
+  version := "0.1",
+  crossPaths := false
+)
+
+lazy val module1 =
+  project
+    .settings(commonSettings)
+    .settings(
+      libraryDependencies ++= Seq(
```

Comment:
Can we reduce the number of dependencies to just one to simplify the test case?

Reply:
Yes I'll do that. As you note, I started with a copy of another test